source/backends/debian/debpkgindex.d

```diff
@@ -63,65 +63,114 @@ public:
         indexChanged = null;
     }
 
-    private void loadPackageLongDescs (DebPackage[string] pkgs, string suite, string section)
+    private immutable(string[]) findTranslations (const string suite, const string section)
     {
-        immutable enDescPath = buildPath ("dists", suite, section, "i18n", "Translation-en.%s");
-        string enDescFname;
+        import std.regex : ctRegex, matchFirst;
+
+        immutable inRelease = buildPath (rootDir, "dists", suite, "InRelease");
+        auto regex = ctRegex!(r"Translation-(\w+)$");
+        auto ret = appender!(string[]);
 
         try {
-            enDescFname = downloadIfNecessary (rootDir, tmpDir, enDescPath);
-        } catch (Exception e) {
-            logDebug ("No long descriptions for %s/%s: %s", suite, section, e.msg);
-            return;
+            const inReleaseContents = getFile (inRelease);
+
+            foreach (const ref entry; inReleaseContents) {
+                auto match = entry.matchFirst(regex);
+
+                if (match.empty)
+                    continue;
+
+                if (!ret.data.canFind (match[1]))
+                    ret.put (match[1]);
+            }
+        } catch (Exception ex) {
+            logWarning ("Couldn't download %s, will assume 'en' is available.", inRelease);
+            return ["en"];
         }
 
-        auto tagf = new TagFile ();
-        tagf.open (enDescFname);
+        return cast (immutable) ret.data;
```

Comment:
Meh, a cast... I wonder whether `ret.data.idup` would work here, to get a immutable copy.